.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+#### Description
+
+<!-- Note: it is best to make pull requests from a branch rather than from main -->
+
+#### Checklist
+
+<!-- replace [ ] with [x] to select -->
+<!-- (delete not applicable items) -->
+
+- [ ] Code modified for feature
+- [ ] Test suite updated with functionality tests
+- [ ] Test suite updated with negative tests
+- [ ] Documentation updated
+
+
+#### Reviewer's checklist:
+
+- [ ] Any issues marked for closing are addressed
+- [ ] There is a test suite reasonably covering new functionality or modifications
+- [ ] This feature/change has adequate documentation added
+- [ ] No obvious mistakes in the code
```

Comment:
I would drop this one and instead have:
- [ ] Code conform to coding style that today cannot yet be enforced via the check style test

Reply:
Also add
- [ ] Commits have short titles and sensible commit messages

Reply:
Done.